documentation/intro-dylan/source/methods-generic-functions.rst

```diff
@@ -48,7 +48,7 @@ return value (or actually does anything). As in C, each typed parameter
 must have its own type declaration; there's no syntax for saying
 "the last three parameters are all integers".
 
-Functions with variable numbers of parameters include the
+Functions with a variable numbers of parameters include the
```

Comment:
a variable number of

Reply:
Actually I changed it to "Functions with the `#rest` keyword in their parameter list accept a variable number of arguments."  I think this gets causality right, and uses "arguments" rather than "parameters" for those who still believe in that pedantic distinction, like myself. :)